.github/workflows/watchman_tests.yml

```diff
@@ -23,27 +23,16 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
 
-      - name: Install Poetry
-        run: curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
-        shell: bash
-
-      - name: Poetry path
-        run: echo "$HOME/.poetry/bin" >> $GITHUB_PATH
-        shell: bash
-
-      - name: Update production dependencies
-        run: poetry update --no-dev
-        shell: bash
-
-      - name: Install dependencies
-        run: poetry install
+      - name: Install # installing pytest-cov is needed because of pytest.ini
+        run: |
+          pip install .
+          pip install pytest pytest-cov nbval
```

Comment:
How does pip work without setup.py nor requirements.txt?

Reply:
pip uses the `pyproject.toml` file (as stated [here](https://pip.pypa.io/en/stable/reference/pip/#build-system-interface)). `setup.py` is required for installing in editable mode (`-e` option), but for that, we just do `poetry install`.